src/common/classes/ClumpletWriter.cpp

```diff
@@ -437,6 +440,9 @@ void ClumpletWriter::insertBytesLengthCheck(UCHAR tag, const void* bytes, const
 			cur_offset += 4;
 		}
 		break;
+	default:
+		fb_assert(lenSize >= 0 && lenSize <= 2 || lenSize == 4);
```

Comment:
@AlexPeshkoff Are you OK with this change? Or did you mean  for the assert to be elsewhere?

Reply:
Usually I use 
  fb_assert(false);
in such cases :)

Reply:
OK, I just thought it might be helpful to have the condition printed out by the macro, instead of it just printing false ;)